docusaurus.config.ts

```diff
@@ -0,0 +1,141 @@
+import { themes as prismThemes } from "prism-react-renderer";
+import type { Config } from "@docusaurus/types";
+import type * as Preset from "@docusaurus/preset-classic";
+
+const config: Config = {
+  title: "Mercurius",
+  tagline: "Mercurius is a GraphQL adapter for Fastify",
+  favicon: "favicon.ico",
+
+  // Set the production url of your site here
+  url: "https://your-docusaurus-site.example.com",
+  // Set the /<baseUrl>/ pathname under which your site is served
+  // For GitHub pages deployment, it is often '/<projectName>/'
+  baseUrl: "/",
+
+  // GitHub pages deployment config.
+  // If you aren't using GitHub pages, you don't need these.
+  organizationName: "facebook", // Usually your GitHub org/user name.
+  projectName: "docusaurus", // Usually your repo name.
```

Comment:
Might be worth refining these

Reply:
Changed see commit here: https://github.com/mercurius-js/website/pull/1/commits/0aa74bb6346712ec6d7cc5bd5b6ae7c3fa1f3374